ovc/client.go

```diff
@@ -18,6 +18,20 @@ import (
 var (
 	// ErrAuthentication represents an authentication error from the server 401
 	ErrAuthentication = errors.New("OVC authentication error")
+	// ErrNotFound represents a resource not found error from the server 404
+	ErrNotFound = errors.New("Resource not found")
+)
+
+// ResponseTimeout for specific api requests
+type ResponseTimeout int
+
+const (
+	// ModelActionTimeout is used for actions that only interfere with the model in the G8
+	ModelActionTimeout ResponseTimeout = 60
```

Comment:
golang has the conectp of time duration see https://golang.org/pkg/time/#Duration
it would probably make more sense to use it here
1 * time.Minute
10 * time.Minute

Reply:
golang says no

![image](https://user-images.githubusercontent.com/15246294/76087608-c9a5e300-5fb6-11ea-8dc9-624dcd42d88d.png)

Reply:
The type definition for `RequestType` could either be dropped completely, replacing all uses of it with `time.Duration` (or a type alias). Otherwise, if the strong type is to be kept, this should work:
```
// ResponseTimeout for specific api requests
type ResponseTimeout time.Duration

const (
	// ModelActionTimeout is used for actions that only interfere with the model in the G8
	ModelActionTimeout = ResponseTimeout(1 * time.Minute)
	// OperationalActionTimeout is used for actions that tamper with deploy resources
	OperationalActionTimeout = ResponseTimeout(10 * time.Minute)
	// DataActionTimeout is used for actions that involve moving data
	DataActionTimeout = ResponseTimeout(24 * time.Hour)
)
```